core/src/test/java/com/opensymphony/xwork2/ognl/SecurityMemberAccessTest.java

```diff
@@ -931,6 +981,15 @@ public void packageInclusion_subclass_both() throws Exception {
     private static String formGetterName(String propertyName) {
         return "get" + propertyName.substring(0, 1).toUpperCase() + propertyName.substring(1);
     }
+
+    @SuppressWarnings("unchecked")
+    private static <T> T mockHibernateProxy(T originalObject, Class<T> proxyInterface) {
```

Comment:
Mocking a Hibernate proxy isn't perfect - integration/acceptance tests would offer better protection against regressions but I'd prefer not to complicate the tests further by introducing a Hibernate session factory etc